### PR TITLE
Additional accessibility updates, docs cleanup

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: install-and-configure/install/helm-install-params.md [hunk 1]
```diff
@@ -1,39 +1,42 @@
...
+Then run your install command:
+
+```
+$ helm install kubecost cost-analyzer \
+    --repo https://kubecost.github.io/cost-analyzer/ \
+    --namespace kubecost --create-namespace \
+    --values values.yaml
+```
+
+## Method 3: Use [_values.yaml_](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml) from the Kubecost Helm chart repository.
```
Comment (srpomeroy) [inline install-and-configure/install/helm-install-params.md:38]: Do we need to keep this? The referenced values file contains the default values applied when running a helm install. You don't need to refer to that file specifically unless the user has modified it.

**Review by srpomeroy — CHANGES_REQUESTED** [review]

File: install-and-configure/install/helm-install-params.md (hunk 1 above)
Comment (srpomeroy) [reply install-and-configure/install/helm-install-params.md:38]: @jessegoodier @chipzoller What are your thoughts?

File: install-and-configure/install/helm-install-params.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ For example, you can only pass a product key if that is all you need to configur
 $ helm install kubecost cost-analyzer \
     --repo https://kubecost.github.io/cost-analyzer/ \
     --namespace kubecost --create-namespace \
-    --set kubecostProductConfigs.productKey.key="123"
+    --set kubecostProductConfigs.productKey.key="123" \
+    --set kubecostProductConfigs.productKey.enabled=true
     ...
-```
```
Comment (srpomeroy) [inline install-and-configure/install/helm-install-params.md]: Looks like the suggestion removed the closing back ticks.

Comment (bstuder99) [reply install-and-configure/install/helm-install-params.md]: Fixed

**Pull request merged**